config/application.rb

```diff
@@ -61,7 +61,10 @@ class Application < Rails::Application
 
     config.email_domain = ENV.fetch('EMAIL_DOMAIN', 'localhost')
 
-    config.max_threads = ENV.fetch('RAILS_MAX_THREADS', 15)
+    # right now we have 4 puma worker with each 5 threads
```

Comment:
I don't think this is right.

The connection pool is per worker, so each worker only needs a connection pool as big as the number of threads is running.